plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1007,6 +1008,18 @@ public void testNullSensitiveTopNPushdown()
         }
     }
 
+    @Test
+    public void testPredicateModuloPushdown()
+    {
+        if (!hasBehavior(SUPPORTS_PREDICATE_MODULO_PUSHDOWN)) {
+            return;
+        }
+        //modulo over bigint
```

Comment:
style: we put a space after `//` comment start

Reply:
removed